src/database/migrations/20190708134018-create-article.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Articles', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      title: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      description: {
+        type: Sequelize.STRING
+      },
+      content: {
+        type: Sequelize.TEXT
+      },
+      image: {
+        type: Sequelize.STRING
+      },
+      slug: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      publishedDate: {
+        type: Sequelize.DATE,
+        field: 'published_date'
+      },
+      status: {
+        type: Sequelize.ENUM('draft', 'active', 'deactivated'),
+        // type: Sequelize.ENUM('DRAFT', 'ACTIVE', 'DEACTIVATED'),
```

Comment:
I guess you forgot to remove this @tobechukwuobitube

Reply:
Yes, thanks for the reminder. Done that already